shared/naturalcrit/markdown.js

```diff
@@ -238,7 +268,38 @@ const definitionLists = {
 	}
 };
 
-Marked.use({ extensions: [mustacheSpans, mustacheDivs, mustacheInjectInline, definitionLists] });
+let pageBlockTopLevel = true;
+let pageBlockNumber = 0;
+const pageBlocks = {
+	name  : 'pageBlock',
+	level : 'block',
+	start(src) { return pageBlockTopLevel; },
+	tokenizer(src, tokens) {
+		if(!pageBlockTopLevel) return false;
+		const pageArray = src.split(/^\\page$/gm);
```

Comment:
Since each token just needs to get the next page between `\page`, we can put a limit on how much splitting occurs to save time:

```suggestion
		const pageArray = src.split(/^\\page$/gm,1);
```

Reply:
May need to be 2 depending how `split` works.

Reply:
Applied.